### ISSUE TITLE
Dimension specific versions of widgets.png?

### ISSUE THREAD

**Issue by Regnander** [sp614x/optifine#1123]
Is it possible to have a different version of `widgets.png` for each dimension, kind of like the Custom Loading Screens feature?

**Comment by realjackasterisk** [comment]
Why not just do this for all GUI textures?

**Comment by Regnander** [comment]
Good point, even though _all_ GUI textures aren't that interesting, like `demo_background.png`, `resource_packs.png`, `server_selection.png` and `world_selection.png`.
